Absolute paths don't work in --custom-formatter argument of CLI

Unlike the `--config` argument, the `--custom-formatter` argument only works with relative paths:

```
$ stylelint --custom-formatter=/var/data/formatter.js foo.css

module.js:327
    throw err;
    ^
Error: Cannot find module '/home/username/var/data/formatter.js'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/lib/node_modules/stylelint/dist/cli.js:61:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
```

--- a/src/cli.js
+++ b/src/cli.js
@@ -115,7 +115,7 @@ const cli = meow(meowOptions, minimistOptions)
 
 let formatter = cli.flags.formatter
 if (cli.flags.customFormatter) {
-  let customFormatter = path.isAbsolute(cli.flags.customFormatter)
+  const customFormatter = path.isAbsolute(cli.flags.customFormatter)
     ? cli.flags.customFormatter
     : path.join(process.cwd(), cli.flags.customFormatter)
   formatter = require(customFormatter)

--- a/src/cli.js
+++ b/src/cli.js
@@ -115,9 +115,9 @@ const cli = meow(meowOptions, minimistOptions)
 
 let formatter = cli.flags.formatter
 if (cli.flags.customFormatter) {
-  var customFormatter = path.isAbsolute(cli.flags.customFormatter)
+  let customFormatter = path.isAbsolute(cli.flags.customFormatter)
     ? cli.flags.customFormatter
-    : path.join(process.cwd(), cli.flags.customFormatter);
+    : path.join(process.cwd(), cli.flags.customFormatter)
   formatter = require(customFormatter)
 }
 

--- a/src/cli.js
+++ b/src/cli.js
@@ -115,7 +115,10 @@ const cli = meow(meowOptions, minimistOptions)
 
 let formatter = cli.flags.formatter
 if (cli.flags.customFormatter) {
-  formatter = require(path.join(process.cwd(), cli.flags.customFormatter))
+  var customFormatter = path.isAbsolute(cli.flags.customFormatter)
+    ? cli.flags.customFormatter
+    : path.join(process.cwd(), cli.flags.customFormatter);
+  formatter = require(customFormatter)
 }
 
 const optionsBase: Object = {

